aqueduct/metrics/collect.py

```diff
@@ -30,13 +30,15 @@ def __init__(self):
         self.queue_sizes = MetricsItems()
         self.tasks_stats = TasksStats()
         self.memory_usage = MetricsItems()
+        self.dead_processes_count = MetricsItems()
```

Comment:
word 'count' in `dead_processes_count` seems like unnecessary, it will be simplier id you make just `self.dead_processes`

Reply:
Now it is `processes_stats`